apps/pigment-css-next-app/next.config.js

```diff
@@ -104,6 +111,26 @@ const pigmentOptions = {
   transformLibraries: ['local-ui-lib'],
   sourceMap: true,
   displayName: true,
+  overrideContext: (context) => {
```

Comment:
A workaround for https://github.com/mui/pigment-css/issues/10

Reply:
Adding this here fixes the issue? Should this then be made part of the library itself?

Reply:
Yes, I can confirm that this prevents the error. I added here just to be a workaround but if you think it should be a part of Pigment, I am happy to add it. However, I think it should be a separate package for integrating with MUI system, not a part of nextjs/vite plugin.

Reply:
If this would be required for every project that uses Material UI and Pigment CSS, we should make it part of the library. Do we know why it is required tough? Maybe we are fixing a symptom, not the problem

Reply:
related issue: https://github.com/mui/pigment-css/issues/8